### ISSUE TITLE
Adding hashtags to feeds without following them

### ISSUE THREAD

**Issue by SuperFola** [TheBLVD/mammoth#561]
### Is your feature request related to a problem? Please describe.

I’m frustrated when hashtags are flooded with messages, and it ends up cluttering my home feed. However I still like to check on those hashtags from time to time!

### Describe the solution you'd like.

I’d like to create feeds from one or more hashtags, without having to follow them and have their posts in my home feed too.

### Describe alternatives you've considered.

I tried creating a list but Mastodon’s lists only allow you to put people in them, not hashtags.

**Comment by derspyy** [comment]
that's a great idea :) 
i'm wondering how we'd be able to do this without making it a bit confusing, or conflicting with mastodon's hashtag follow. we'd have to either allow you to pin the hashtag to the carousel without following it, or allow you to filter out hashtags from the home timeline without unfollowing them :P

**Comment by SuperFola** [comment]
I've been thinking about this for the past days, I think it can be summed up as pinning a hashtag (without following it), something you can already do in the advanced mastodon UI.

It's related to the advanced interface and those aren't exported anywhere it seems (eg in api responses and stuff). Megalodon also has this feature iirc, perhaps more clients, but I'm unsure about how it works with mastodon pinned hashtags.

**Comment by derspyy** [comment]
i agree that this is the way to go. will certainly be experimenting 😝
